mava/components/building/distributor.py

```diff
@@ -37,6 +37,7 @@ class DistributorConfig:
     single_process_max_episodes: Optional[int] = None
     is_test: Optional[bool] = False
     wait: Optional[bool] = False
+    metrics_checkpoint: Tuple = ("mean_episode_return",)
```

Comment:
Why do we save mean_episode_return and not just the best parameters? Is it use in the evaluation somehow?

Reply:
This because we can checkpoint the best performance depending on multiple metrics (e.g. best network for win rate it can be not the best network for the return)